docs/quickstart-aws.md

```diff
@@ -9,7 +9,7 @@ As a result, you'll get Kubernetes 1.14.1 High-Available (HA) clusters with thre
 To follow this quick start, you'll need:
 
 * `kubeone` v0.7.0 or newer installed, which can be done by following the `Installing KubeOne` section of [the README](https://github.com/kubermatic/kubeone/blob/master/README.md),
-* `terraform` v0.11 installed. The binaries for `terraform` can be found on the [Terraform website](https://www.terraform.io/downloads.html)
+* `terraform` v0.12 installed. The binaries for `terraform` can be found on the [Terraform website](https://www.terraform.io/downloads.html)
 
 **Note:** Due to breaking changes made in Terraform v0.12, it's currently not possible to use example Terraform scripts with Terraform v0.12.
```

Comment:
We should probably remove this note from all files or modify it to note that you can't use Terraform older than v0.12.

Reply:
Oh sorry, overlooked it. I could modify the prerequisite in all examples to something like "terraform v0.12 or later installed. Examples do not work with v0.11 and older" Would that be good?

Reply:
Yes, that sounds great.

Reply:
Done

Reply:
Can you just remove the note that says scripts are not compatible with Terraform 0.12? Otherwise looks great. :100:

Reply:
Urg, forgot to push it, sorry :D